tui/installer/screens.py

```diff
@@ -941,7 +942,7 @@ def nsvalue(answers, id):
                 answers['manual-nameservers'][1].append(ns2_entry.value())
                 if ns3_entry.value() != '':
                     answers['manual-nameservers'][1].append(ns3_entry.value())
-            if 'net-admin-configuration' in answers and answers['net-admin-configuration'].isStatic():
+            if 'net-admin-configuration' in answers and answers['net-admin-configuration'].valid() and not answers['net-admin-configuration'].isDHCP():
```

Comment:
This check is now complex enough to warrant pulling it out as a well named method.